script/Arcueid/Day 7/07_00_ARC07_3_1_4.txt

```diff
@@ -122,13 +122,13 @@ I can hardly believe that the being before my eyes right now is the same vampire
 [sha:6ded6007d5d24921c6452f885fdba6b959b4f077]{
 -- Page 630, Offset 18165.
 -- というかこのドレスでもう一度遠野邸に来てくれないかな絶対絵になるから。
-She should come back to the mansion one more time now that she has this dress on.
+She should come back to the mansion now that she has this dress on.
  That would certainly be a sight to behold.
 }
 [sha:aaf196313f817569c6c0ec67098f6cce1be0677e]{
 -- Page 631, Offset 18166.
 -- 「これ、わたしの選択ミスだから、あんまり好きじゃないんだけど。
-"I made a mistake in choosing this dress,
+"I made a mistake in choosing this dress, //Would it be closer to "while making this dress"? Seeing as the previous line seems like she just improvised and created this heatproof dress on the spot rather than she's constantly had it.
```

Comment:
I think that a mistake *while* making makes it sound like the dress doesn't live up to the design, but what she's actually saying here is "selecting/choosing this [dress] was a bad choice"